tests/acceptance/test_bootstrap_artifact.py

```diff
@@ -62,7 +62,7 @@ def img_builder():
                 'MENDER_ARTIFACT_VERIFY_KEY = "%s"'
                 % os.path.join(os.getcwd(), signing_key("RSA").public),
                 'SYSTEMD_AUTO_ENABLE:pn-mender-client = "enable"',
-                'SYSTEMD_AUTO_ENABLE:pn-mender-update = "enable"',
+                'SYSTEMD_AUTO_ENABLE:pn-mender = "enable"',
```

Comment:
Q: Are you sure this will work with the new client? I think this is just a dud in this instance.